server/functions/dolt_procedures.go

```diff
@@ -39,19 +39,27 @@ func initDoltProcedures() {
 		}
 
 		funcVal := reflect.ValueOf(procDef.Function)
-		callable := callableForDoltProcedure(p, funcVal)
+		callable1 := callable1ForDoltProcedure(p, funcVal)
+		callable0 := callable0ForDoltProcedure(funcVal)
```

Comment:
How about calling the functions `noArgCallableForDoltProcedure` and `varArgCallableForDoltProcedure`? That would be more self-describing than callable1 and callable0.

Reply:
Great idea! I'm not good with names at all lol